SUI/SUI/ContentView.swift

```diff
@@ -7,18 +7,166 @@
+        static let buttonImageHeight = 50
+        static let buttonImageCornerRadius = 10
+        static let secondsInHour = 3600
+        static let secondsInMinute = 60
+        static let playImageName = "play.fill"
+        static let stopImageName = "stop.fill"
+        static let backwardImageName = "backward.fill"
+        static let forwardImageName = "forward.fill"
+        static let downloadButtonLabel = "Скачать"
+        static let downloadAlertText = "Скачать трек?"
+        static let shareButtonLabel = "Поделиться"
+        static let yesButtonText = "Да"
+        static let noButtonText = "Нет"
+        static let emptyString = ""
+        static let sliderMinimumValueText = "0:0"
+    }
+    
+    // MARK: - Body
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            setupSongImage
+            setupDurationSlider
+            setupSongNameLabel
+            setupPlayAndStopButtons
+            setupNextAndPreviosTrackButtons
+            setupDownloadAndShareButtons
+        }
+        .padding()
+    }
+    
+    // MARK: - ObserverObject private properties
+    @ObservedObject private var playerView = PlayerViewModel()
+    
+    // MARK: - State private properties
+    
+    @State private var progress: Float = 0
+    @State private var isSaveTrackButtonTapped = false
+    @State private var isShareTrackButtonTapped = false
+    
+    // MARK: - Private properties
+    
+    private var setupSongImage: some View {
+        Image(playerView.getTrackImage())
+            .resizable()
+            .frame(width: CGFloat(Constants.trackImageWidth),
+                   height: CGFloat(Constants.trackImageHeight))
+            .scaledToFit()
+            .cornerRadius(CGFloat(Constants.trackImageCornerRadius))
+    }
+    
+    private var setupDurationSlider: some View {
+         Slider(value: Binding(get: {
+            Double(playerView.playerCurrentTime ?? Float(Constants.floatDefaultValue))
+        }, set: { newValue in
+            playerView.playerCurrentTime = Float(newValue)
+            playerView.setTime(value: Float(playerView.playerCurrentTime ?? Float(Constants.floatDefaultValue)))
+        }), in: 0...Double(playerView.maxDuration)) {
+        } minimumValueLabel: {
+            Text(Constants.sliderMinimumValueText)
+        } maximumValueLabel: {
+            Text("\((playerView.maxDuration % Constants.secondsInHour) / Constants.secondsInMinute):\((playerView.maxDuration % Constants.secondsInHour) % Constants.secondsInMinute)")
         }
         .padding()
     }
+    
+    private var setupSongNameLabel: some View {
+        Text(playerView.getTrackName())
+    }
+    
+    private var setupPlayAndStopButtons: some View {
+          HStack {
+            Button {
+                playerView.play(trackName: playerView.getTrackName())
```

Comment:
```suggestion
                viewModel.play(trackName: playerView.getTrackName())
```
 playerView.play(trackName: playerView.getTrackName())
нарушает mvvm